src/rest/openapi.yaml

```diff
@@ -0,0 +1,455 @@
+openapi: 3.0.3
```

Comment:
This is a feature we wanted long ago, thanks @agners! 👍

Reply:
You are welcome. It is really useful for testing, I quickly discovered some bugs/issues/discrepancies once I had the description written :smile: